pkgs/development/python-modules/requests-futures/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, requests
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "requests-futures";
+  version = "0.9.9-27-gf126048";
```

Comment:
From where did you get the version number?

Reply:
`git describe` minus the leading `v`